### ISSUE TITLE
Choose a face does not work

### ISSUE THREAD

**Issue by RaisingAwesome** [wladradchenko/wunjo.wladradchenko.ru#34]
In Wunjo AI, when I use the tools for fencing off a face such as the dot/line too, lines are not added to the screen. In turn, when processed it throws an error due to not having a face region selected.

Update: When I click a single point in the middle of the face, it does execute correctly.

**Comment by wladradchenko** [comment]
Hello. Inside is not tool as line, just point. This is not line tool, that changes between set point or move view by holding mouse.You need to set point. This is not bug.

**Comment by MadeByKit** [comment]
i think he needs to make a new tutorial video on the new software cause i also dont understand

**Comment by wladradchenko** [comment]
Hi. This has already been implemented.  

1) The user can disable automatic loading of the model.  

2) When opening modules, there is a message that not all models, if you click on it, it will be open where to download the models and where to put them.  

3) There is documentation wiki on all issues, for example how to transfer the program to another drive, etc., and a smart bot in [Telegram](https://t.me/wladblog/65) if you are too lazy to read.  

4) I recorded a video on how to create a build application, it is available in two languages on [my channel](https://youtu.be/00gzOC_cw9Y?feature=shared). There is also a demonstration about downloading models manually. In app u will able to find all videos about app. If u wanna more videos about Wunjo or Neural Network Trains, just share with video to your friends and write comments under video about your wishes.
